src/providers/details/tabs/Hosts/components/utils/onSaveHost.ts

```diff
@@ -0,0 +1,156 @@
+import { Base64 } from 'js-base64';
```

Comment:
why u need it if u have atob and btoa ?

Reply:
I am not sure how it handles ip addresses though - read it's not always supported - so I prefer leave it as is

Reply:
what do u mean not supported? did u tried?